### PR TITLE
MNG-88_StartButtonActivation

### PR DESCRIPTION
**Pull request by mariusz-sm**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 <body>
 
-<div id="app">
+<!-- <div id="app">
```
Comment (lukaszdutka) [inline index.html]: why those comments?

File: index.html [hunk 2]
```diff
@@ -23,9 +23,9 @@
         TypeScript Mangos project for CodersCamp2020
     </footer>
 
-</div>
+</div> -->
 
-    <template id="starting-page-template">
+    <!-- <template id="starting-page-template"> -->
```
Comment (lukaszdutka) [inline index.html]: and here (why those comments?)

File: src/index.ts [hunk 3]
```diff
@@ -1,4 +1,14 @@
-//Uncomment to see the choose pokemon page
-//const newTemplate = document.querySelector('#choose-page-template')!;
-//const app = document.querySelector('#app')!;
-//app.innerHTML = newTemplate.innerHTML
\ No newline at end of file
+import { activateStart } from './activateStart';
```
Comment (lukaszdutka) [inline src/index.ts:2]: it seems like you didn't commited/pushed `activateStart` file. Also, remember about extension (.ts) :)

File: index.html (hunk 1 above)
Comment (mariusz-sm) [reply index.html]: I commented it to see template for starting screen. Now it is corrected.

File: src/index.ts (hunk 3 above)
Comment (mariusz-sm) [reply src/index.ts:2]: activateStart.ts file was added.

Should I add .ts extension when I import file? It works without extension but if it is not proper I will change it.

File: src/activateStart.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// import { config } from '../styles/SASS/config.scss'
```
Comment (mariusz-sm) [inline src/activateStart.ts]: I tried to import variables from config.scss. For this purpose I appiled following schame but it does not work
[https://til.hashrocket.com/posts/sxbrscjuqu-share-scss-variables-with-javascript](url)

@lukaszdutka Do you know how to use it with TS?

Comment (lukaszdutka) [reply src/activateStart.ts]: It looks like it needs React. I don't know if it is possible to share variables to javascript, but:
workaround (or even better approach) is to apply different style class (i.e. for button: `button` and `button-disabled`) and changing classes in code is more appropriate :)

File: src/index.ts (hunk 3 above)
Comment (lukaszdutka) [reply src/index.ts:2]: my bad, it's ok :) sorry

File: src/activateStart.ts (hunk 4 above)
Comment (lukaszdutka) [reply src/activateStart.ts]: See here: https://clubmate.fi/javascript-adding-and-removing-class-names-from-elements

Comment (mariusz-sm) [reply src/activateStart.ts]: Done :)

**Review by lukaszdutka — APPROVED** [review]
looks good :)

**Pull request merged**